src/Erl/Process.purs

```diff
@@ -10,10 +10,10 @@ module Erl.Process
   , module RawExport
   ) where
 
-import Prelude
+import Prelude (class Eq, Unit, eq, (<$>))
```

Comment:
Rather keep `Prelude` imported implicitly, it is other imports which should be explicitly qualified (in this case `Control.Monad.Eff` should be